tests/unittest_modutils.py

```diff
@@ -72,9 +73,9 @@ def test_find_egg_module(self) -> None:
             ["data", "MyPyPa-0.1.0-py2.5.egg", self.package],
         )
 
-    def test_find_distutils_submodules_in_virtualenv(self) -> None:
-        found_spec = spec.find_spec(["distutils", "version"])
-        self.assertEqual(found_spec.location, distutils.version.__file__)
+    def test_find_wrapt_submodules_in_virtualenv(self) -> None:
+        found_spec = spec.find_spec(["wrapt", "decorators"])
+        self.assertEqual(found_spec.location, wrapt.decorators.__file__)
```

Comment:
The test was specifically designed to test the location of `distutils`. So this change wouldn't fix it